### DISCUSSION TITLE
`LOW_BANDWIDTH` regression

### DISCUSSION THREAD

**Discussion by jef** [jef/streetmerchant#1079] [General]
### Description

Currently in some conditions, using `LOW_BANDWIDTH="true"` will cause pages to not load required selectors. This means that pages may be reporting out of stock when in fact _could be_ in stock.

### Possible solution

- Reevaluate what to disable in `LOW_BANDWIDTH` mode
- Allowlist of certain images

**Comment by Vuand** [comment]
The program uses ridiculous amounts of data and providers usually only allow a max of 1.2 Tb of data per month. Which sites would be affected by using low bandwidth mode?

**Comment by jef** [comment]
> The program uses ridiculous amounts of data and providers usually only allow a max of 1.2 Tb of data per month. Which sites would be affected by using low bandwidth mode?

I saw this with `bestbuy` so far. There could be others. Although, this should ideally get fixed.

**Comment by Vuand** [comment]
So, even with the low bandwidth enabled, your saying that there a way to get around the false negatives even if images and part of the pages aren't going to be loaded. Also how soon do you think before it will take to be fixed and ideally data and bandwidth would it be saving per day?

**Comment by MRizkBV** [comment]
I saw Amazon affected by it a few times. The entire right side of the page where the add to cart button is, and seller details would just be cut off.
